site/blog/2022-10-27-clean-your-aws-account.mdx

```diff
@@ -0,0 +1,108 @@
+
+Having unknown resources running will cause you to incur on extra costs, or could cause unexpected conflicts. Cleaning those manually can be a hard task... manually visiting each AWS service page in the console, investigating the dependencies, manually cleaning... it is a time-consuming, error-prone task.
+
+To solve this problem you can use our IaSQL engine. By simply clicking a button, our engine will get an snapshot of the status of your cloud, showing you all the resources that you have created over the time, on multiple regions.
+There may be some resources that you are aware of and you need to keep, but also some leftovers or unknown resources created by some automated process.
+
+**Want to give a try? Join our service for free and start taking control of your cloud.**
+
+<!--- https://www.urlencoder.org/ -->
+
+<p>
+  <a href="https://app.iasql.com/#/button/IaSQL_AWS_account_snapshot/--%20Installing%20all%20IaSQL%20modules%0ASELECT%20iasql_install%28variadic%20array%28select%20module_name%20from%20iasql_modules_list%28%29%29%29%3B%0A%0A--%20iam%0A--select%20%2A%20from%20iam_user%3B%0A--select%20%2A%20from%20iam_role%3B%0A--%20ec2%0A--select%20%2A%20from%20instance%3B%0A--select%20%2A%20from%20security_group%3B%0A--select%20%2A%20from%20elastic_ip%3B%0A--select%20%2A%20from%20general_purpose_volume%3B%0A--%20network%0A--select%20%2A%20from%20vpc%3B%0A--select%20%2A%20from%20subnet%3B%0A--select%20%2A%20from%20nat_gateway%3B%0A--select%20%2A%20from%20endpoint_gateway%3B%0A--select%20%2A%20from%20endpoint_interface%3B%0A--select%20%2A%20from%20target_group%3B%0A--select%20%2A%20from%20load_balancer%3B%0A--%20ecs%0A--select%20%2A%20from%20service%3B%0A--select%20%2A%20from%20cluster%3B%0A--%20code%2A%0A--select%20%2A%20from%20codebuild_project%3B%0A--select%20%2A%20from%20distribution%3B%0A--select%20%2A%20from%20listener%3B%0A--select%20%2A%20from%20container_definition%3B%0A--select%20%2A%20from%20repository%3B%0A--%20route%2053%0A--select%20%2A%20from%20hosted_zone%3B%0A--%20s3%0A--select%20%2A%20from%20bucket%3B%0A--%20rds%0A--select%20%2A%20from%20rds%3B%0A--%20dynamo%0A--select%20%2A%20from%20dynamo_table%3B%0A--%20acm%0A--select%20%2A%20from%20certificate%3B%0A--%20cloud%20watch%0A--select%20%2A%20from%20log_group%3B%0A--%20secrets%20and%20credentials%0A--select%20%2A%20from%20secret%3B%0A--select%20%2A%20from%20source_credentials_list%3B%0A--%20graphql%0A--select%20%2A%20from%20graphql_api%3B%0A--%20lambda%0A--select%20%2A%20from%20lambda_function%3B%0A--%20elasticache%0A--select%20%2A%20from%20memory_db_cluster%3B%0A--select%20%2A%20from%20cache_cluster%3B%0A--%20api%20gateway%0A--select%20%2A%20from%20api%3B">
+    <button class="button button--lg button--primary">List AWS resources</button>
+  </a>
+</p>
+
+```
```

Comment:
tag this snippet as SQL and also put it before the button like in the docs? https://raw.githubusercontent.com/iasql/iasql-engine/main/site/docs/sample-queries/aws_ec2.md